hypha/apply/api/v1/views.py

```diff
@@ -61,6 +61,19 @@ def get_queryset(self):
             Prefetch('reviews', Review.objects.submitted()),
         )
 
+    @action(detail=True, methods=['put'])
+    def set_summary(self, request, pk=None):
```

Comment:
@vimal1083, would be good to have a separate serializer to validate the post data that is coming. As it is only for one field.

```python
class SubmissionSummarySerializer(serializers.Serializer):
     summary = serializers.CharField(write_only=True)
```
And then in `set_summary` action:

```python
...
ser = SubmissionSummarySerializer(data=request.data)
ser.is_valid(raise_exception=True)
summary = ser.validated_data['summary']
submission.summary = summary
submission.save()
ser = self.get_serializer(submission)
return Response(ser.data)
```